app/src/main/java/ml/docilealligator/infinityforreddit/activities/PostGalleryActivity.java

```diff
@@ -244,11 +244,7 @@ public void getItemOffsets(@NonNull Rect outRect, @NonNull View view, @NonNull R
                 int halfOffset = offset / 2;
 
                 if (nColumns == 2) {
-                    if (spanIndex == 0) {
-                        outRect.set(halfOffset, 0, halfOffset, offset);
-                    } else {
-                        outRect.set(halfOffset, 0, halfOffset, offset);
-                    }
+                    outRect.set(halfOffset, 0, halfOffset, offset);
```

Comment:
You stopped half way there - all the branches are the same and `nColumns` is always `2` or `3`

Reply:
Done